test/com/pty4j/FailedToLoadNativePtyLibTest.java

```diff
@@ -0,0 +1,21 @@
+package com.pty4j;
+
+import junit.framework.TestCase;
+import org.junit.Assert;
+import testData.RepeatTextWithTimeout;
+
+import java.io.IOException;
+
+public class FailedToLoadNativePtyLibTest extends TestCase {
+  public void testErrorMessage() {
+    String[] cmd = TestUtil.getJavaCommand(RepeatTextWithTimeout.class, "2", "1000", "Hello, World");
+    boolean failed = false;
+    try {
+      PtyProcess.exec(cmd);
+    }
+    catch (IOException e) {
+      failed = true;
```

Comment:
It would be nicer to use JUnit4 and syntax like: 
@Test(expected = IOException.class)